tests/console/salt.pm

```diff
@@ -41,7 +41,9 @@ sub run {
     }
 
     quit_packagekit;
-    zypper_call('in salt-master salt-minion');
+    my @packages = qw(salt-master);
+    push @packages, 'salt-minion' unless is_jeos;
```

Comment:
A few lines later we sed a minion config file - which went work if salt-minion is not installed

See new TW/JeOS failure
https://openqa.opensuse.org/tests/3603782#step/salt/2

Reply:
On TW, salt is (intentionally) not installed, so this should probably be an `is_jeos && (is_sle || is_leap)`